### ISSUE TITLE
508-defect-2: Error messages should only appear onblur

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#53475]
### Point of contact

joshkimux

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

Currently on edit pages, when making a change to a field, errors emerge immediately character by character while inputting into the field which is verbose and distracting to screen reader users.

### Reproduction steps

1. Turn on your screen reader. 
2. On any of the edit pages, start backspacing to clear out the input field.
3. Confirm an error is thrown immediately when a change is made and not on blur.

### Proposed solution or next steps

According to the design system, in all cases, only show error validation messages or stylings after a user has interacted with a particular field. Setting errors to appear on blur should do the trick.

That being said, there's an [open ticket exploring on submit validation](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1130) as opposed to on blur.

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [X] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by saderagsdale** [comment]
@joshkimux Can you post the supplemental claim pages that this appears on?

**Comment by saderagsdale** [comment]
@Mottie to review in staging and identify places where this is happening.

**Comment by joshkimux** [comment]
@saderagsdale @Mottie yes! I had this on all of the edit pages e.g.
- Edit home phone number
- Edit mobile phone number
- etc

When backspacing an existing home phone number, the error would appear (if my memory serves me correctly). Can double check on this and add a video when my meetings wrap up today 🙏

**Comment by saderagsdale** [comment]
@adamwhitlock1 Hey Adam - can you confirm if you're working on this one too?

**Comment by adamwhitlock1** [comment]
@saderagsdale I'm not sure of this issue as being resolvable without changes to the form system itself. We trigger focus onto the field when the edit button is clicked as per accessibility recommendations in that regard, so the validation starts firing once any input is done.

I will have to look into this, and can see if this can be added to the next sprint, as I will not have time for this work during this sprint.

**Comment by Mottie** [comment]
@saderagsdale We've got a fix PR ready, so we won't need to wait too much longer 😸

**Comment by Mottie** [comment]
Fixed and available for testing in staging

**Comment by joshkimux** [comment]
https://user-images.githubusercontent.com/14154792/222446935-07aafc9b-564f-4218-b504-566c3398ed43.mov

@adamwhitlock1 and @Mottie y'all are wizards; have no idea how y'all fixed this but it's working ❤️
